src/styles/reviews.scss

```diff
@@ -0,0 +1,39 @@
+.card__reviews {
+  margin-top: 7px;
+  align-items: center;
+  text-align: center;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.card__reviews-stars-4 {
+  display: flex;
+}
```

Comment:
it's not the separate BEM block, you must have only stars and card block (and two block files)